packages/monitors/src/CRMonitor.js

```diff
@@ -168,13 +183,13 @@ class CRMonitor {
           "% which is below the " +
           wallet.crAlert * 100 +
           "% threshold. Current value of " +
-          this.empProps.syntheticCurrencySymbol +
```

Comment:
I somewhat changed the implementation here. Previously we were logging the "Current value of" the symbol but it really should be the identifier, not the symbol, that is included in this log.